commit.go

```diff
@@ -244,7 +244,8 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 	}
 	if exporting {
 		// Copy everything.
-		err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter))
+		// TOM: todo add credsContext
+		err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter, nil))
```

Comment:
yes I think so too.  I only made the change here now as I added a param to this function.  I plan to come back later and get commit and push fully authenticated.